lib/trailblazer/test/operation/assertions.rb

```diff
@@ -51,5 +53,17 @@ def _assert_call(operation_class, params: raise, user_block: raise, &block)
 
       result
     end
+
+    # @private
+    class TrbHash < Hash
+      include Hashie::Extensions::DeepMerge
+    end
```

Comment:
I can change it 👍🏼

Reply:
I don't think it can be used the way you wanted.
This is their API:
```
module Hashie
  module Extensions
    module DeepMerge
      # Returns a new hash with +self+ and +other_hash+ merged recursively.
      def deep_merge(other_hash, &block)
        copy = dup
        copy.extend(Hashie::Extensions::DeepMerge) unless copy.respond_to?(:deep_merge!)
        copy.deep_merge!(other_hash, &block)
      end
    end
  end
end
```

Reply:
Yeah, I was actually thinking about patching that and PRing since I am not a fan of module inclusions anymore. :wink: